ansible/roles/test/files/ptftests/py3/advanced-reboot.py

```diff
@@ -1387,7 +1387,7 @@ def neigh_lag_status_check(self):
         for neigh in self.ssh_targets:
             flap_cnt = None
             if self.test_params['neighbor_type'] == "sonic":
-                flap_cnt = self.cli_info[neigh][1]
+                flap_cnt = self.cli_info[neigh]['po'][1]
```

Comment:
This an unrelated but necessary fix?

Reply:
Correct, this is an unrelated but necessary fix.

Reply:
Split into #10685.